package.json

```diff
@@ -297,6 +298,7 @@
   },
   "dependencies": {
     "@marp-team/marp-cli": "^1.2.0",
-    "@marp-team/marp-core": "^2.1.0"
+    "@marp-team/marp-core": "^2.1.0",
+    "tmp": "^0.2.1"
```

Comment:
`tmp` may not work correctly due to circular dependency if tried to bundle into the extension by adding to `devDependencies` instead.

Even if used `tmp` as external, package size of extension should not increase because `tmp` is already included into `@marp-team/marp-cli`.